spec/index.bs

```diff
@@ -4941,8 +4941,8 @@ dictionary GPUPrimitiveState {
     GPUFrontFace frontFace = "ccw";
     GPUCullMode cullMode = "none";
 
-    // Enable depth clamping (requires "depth-clamping" feature)
-    boolean clampDepth = false;
+    // Requires "depth-clip-control" feature.
+    boolean unclippedDepth = false;
```

Comment:
u-nit: What about `boolean clipDepth = true`? to avoid the double negatives?

Reply:
We discussed this with the editors, and having "true" defaults is discouraged in Web APIs.

Reply:
> It is strongly suggested not to use a default value of true for boolean-typed arguments, as this can be confusing for authors who might otherwise expect the default conversion of undefined to be used (i.e., false). [API-DESIGN-PRINCIPLES]

I.e. it makes the behavior of `clipDepth: undefined` and `clipDepth: false` different, and that's weird because it's a boolean argument and one might expect it to get cast to a boolean.